### PR TITLE
[ch336] Admin controls for dataset person types

### PR DESCRIPTION
**Pull request by jnu**
- Adds a UI for the admin to manage person types for a dataset
 - Ensures person types get deduped by name on the backend
 - Show all datasets to admin when they login
 - Clean up some more dummy data stuff

### CODE REVIEW

File: api/queries.py [hunk 1]
```diff
@@ -232,3 +233,13 @@ def resolve_organizations(obj, info):
     '''
     session = info.context['dbsession']
     return session.query(Organization).filter(Organization.deleted == None).order_by(Organization.name.asc()).all()
+
+
+@query.field("personTypes")
```
Comment (seerocode) [inline api/queries.py:238]: Person types worked without this field. Why does it need to be sorted in ascending order?

File: client/src/App.css [hunk 2]
```diff
@@ -8,7 +8,7 @@
 }
 
 .ant-layout {
-  min-height: initial;
+  min-height: max-content;
```
Comment (seerocode) [inline client/src/App.css:11]: 👍🏽

**Review by seerocode — APPROVED** [review]

**Review by nmanduj1 — APPROVED** [review]
🚢

File: api/queries.py (hunk 1 above)
Comment (jnu) [reply api/queries.py:238]: this is a new top-level query that fetches all of the personTypes used in the app. this is used for autocomplete in the admin "edit program" screen to help them reuse existing personTypes.

ascending order for two reasons: 1) ensures determinism in tests, and 2) they will probably always be used alphabetically anyway (e.g. in the autocomplete dropdown)

File: client/src/App.css (hunk 2 above)
Comment (jnu) [reply client/src/App.css:11]: looked like there was an optimization (or possibly just a bug) in the minified CSS that replaced `min-height: initial` with `min-height: 0`, which breaks styles. this fixes that.

**Pull request merged**